beginner_source/quickstart/quickstart_tutorial.py

```diff
@@ -0,0 +1,241 @@
+
+    def forward(self, x):
+        x = self.flatten(x)
+        x = F.relu(self.layer1(x))
+        x = F.relu(self.layer2(x))
+        x = self.output(x)
+        return F.softmax(x, dim=1)
+model = NeuralNetwork().to(device)
+    
+print(model)
+
+######################################################################
+# Optimizing Parameters and Training
+# ---------------------
+# 
+# Optimizing model parameters requires a loss function, optimizer,
+# and the optimization loop. 
+# Training a model is essentially an optimization process similar to the one we described in the
+# `Autograd <autograd_tutorial.html>`_ section. We run the optimization process on the whole dataset 
+# several times, and each run is refered to as an **epoch**. During each run, we present data 
+# in **minibatches**, and for each minibatch compute gradients and correct parameters of the model 
+# according to back propagation algorithm. Read more about the `Optimization Loop <optimization_tutorial.html>`_.
+#
+
+# Cost function used to determine best parameters.
+cost = torch.nn.BCELoss()
+
+# This is used to create optimal parameters.
+learning_rate = 1e-3
+optimizer = torch.optim.Adam(model.parameters(), lr=learning_rate)
+
+# Create the training function.
+def train(dataloader, model, loss, optimizer):
+    size = len(dataloader.dataset)
+    for batch, (X, Y) in enumerate(dataloader):
+        X, Y = X.to(device), Y.to(device)
+        optimizer.zero_grad()
+        pred = model(X)
+        loss = cost(pred, Y)
+        loss.backward()
+        optimizer.step()
+    
+        if batch % 100 == 0:
+            loss, current = loss.item(), batch * len(X)
+            print(f'loss: {loss:>7f}  [{current:>5d}/{size:>5d}]')
+
+
+# Create the validation/test function
+
+def test(dataloader, model):
+    size = len(dataloader.dataset)
+    model.eval()
+    test_loss, correct = 0, 0
+
+    with torch.no_grad():
+        for batch, (X, Y) in enumerate(dataloader):
+            X, Y = X.to(device), Y.to(device)
+            pred = model(X)
+
+            test_loss += cost(pred, Y).item()
+            correct += (pred.argmax(1) == Y.argmax(1)).type(torch.float).sum().item()
+
+    test_loss /= size
+    correct /= size
+
+    print(f'\nTest Error:\nacc: {(100*correct):>0.1f}%, avg loss: {test_loss:>8f}\n')
+
+
+# Call the train and test function in a training loop with the number of epochs indicated.
+
+epochs = 5
+
+for t in range(epochs):
+    print(f'Epoch {t+1}\n-------------------------------')
+    train(train_dataloader, model, cost, optimizer)
+    test(test_dataloader, model)
+print('Done!')
+
+######################################################################
+# Saving Models
+# -------------
+# 
+# PyTorch has different ways you can save your model. One way is to serialize the internal model state to a file. Another would be to use the built-in `ONNX <https://github.com/onnx/tutorials>`_ support.
+# Saving PyTorch Model Dictionary
```

Comment:
I think you intended this to be a comment in the code block, but it's part of the markdown. Inserting a linebreak should help!